packages/backend-common/src/sqs.ts

```diff
@@ -87,12 +87,26 @@ export const generateOutputSignedUrlAndSendMessage = async (
 		originalFilename,
 		outputBucketUrls: signedUrls,
 		languageCode,
-		translate,
+		translate: false,
 	};
-	return await sendMessage(client, queueUrl, JSON.stringify(job), s3Key);
+	const messageResult = await sendMessage(
+		client,
+		queueUrl,
+		JSON.stringify(job),
+		s3Key,
+	);
+	if (!isSqsFailure(messageResult) && translationRequested) {
+		return await sendMessage(
+			client,
+			queueUrl,
+			JSON.stringify({ ...job, translate: true }),
+			s3Key,
+		);
```

Comment:
I'm not sure I understand this bit - we put two messages in the queue if the user has requested a translation? Doesn't the worker create the translation as its transcribing?

Reply:
The way translation works at the moment is that we send two messages and the translation runs on one worker, the normal transcription on the other. My idea there was to speed up the transcription/translation - if we do it all on the same worker then it could take twice as long

Reply:
Ah ok thanks, I hadn't realized this.